collections/_article/dev-snapshot-godot-4-5-beta-7.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Dev snapshot: Godot 4.5 beta 7"
+excerpt: Two more for the road!
+categories: [pre-release]
+author: Thaddeus Crews
+image: /storage/blog/covers/dev-snapshot-godot-4-5-beta-7.webp
+image_caption_title: Strange Jigsaws
+image_caption_description: A game by FLEB
+date: 2025-08-28 12:00:00
+---
+
+While we initially anticipated our next snapshot to begin release candidate phase, we left open to the possibility of another beta pass if any significant blockers still persisted. And while the overwhelming majority have since been handled, this extra pass is to ensure debug symbols are supported for Android platforms. While this was always something that could be done when building from source, a long-term goal is official access/downloads for to debug symbols, making this something we want to get right out of the gate.
```

Comment:
```suggestion
While we initially anticipated our next snapshot to begin the release candidate phase, we left open the possibility of another beta pass if any significant blockers persisted. And while the overwhelming majority have since been handled, this extra pass is to ensure debug symbols are supported for Android platforms. While this was always something that could be done when building from source, a long-term goal is official access to debug symbols, making this something we want to get right out of the gate.
```